.github/workflows/ci.yml

```diff
@@ -139,7 +139,7 @@ jobs:
     secrets: inherit
     with:
       os: macos
-      runner: macos-arm
+      runner: macos-latest-xlarge # per https://github.blog/2023-10-02-introducing-the-new-apple-silicon-powered-m1-macos-larger-runner-for-github-actions/
```

Comment:
This needs to be macos-14 to make sure we're not paying for the extra large runners.

Reply:
This is separate commit. I did the change because our dedicated `macos-arm` runner was down, and I couldn't remember whom to bother about it. I'll roll it back, but perhaps we should consider using the free-for-open-source M1 runners on github?

Reply:
Yes, we saw that the arm runner is broken. Since there are free runners on GitHub we should use those.